GeneratorInterface/Herwig7Interface/python/Herwig7_Dummy_ReadLHE_GEN_SIM.py

```diff
@@ -0,0 +1,18 @@
+import FWCore.ParameterSet.Config as cms
+
+generator = cms.EDFilter("Herwig7GeneratorFilter",
+    run = cms.string('InterfaceMatchboxTest'),
+    repository = cms.string('${HERWIGPATH}/HerwigDefaults.rpo'),
+    dataLocation = cms.string('${HERWIGPATH:-6}'),
+    generatorModule = cms.string('/Herwig/Generators/EventGenerator'),
+    eventHandlers = cms.string('/Herwig/EventHandlers'),
+    configFiles = cms.vstring(),
+    crossSection = cms.untracked.double(-1),
+    parameterSets = cms.vstring(),
+    filterEfficiency = cms.untracked.double(1.0),
+    Matchbox = cms.vstring( 'read LHE.in',
+        'set LesHouchesReader:FileName ${CMSSW_BASE}/src/cmsgrid_final.lhe.gz'
```

Comment:
how this file cmsgrid_final.lhe.gz get in to ${CMSSW_BASE}/src/cmsgrid_final.lhe.gz?

Reply:
This is an example config for LHE showering, when you already got your LHEs produced. I was asked back in the day by @jfernan2 to write provide such a config for studies.